grpc/src/main/java/com/linecorp/armeria/internal/grpc/GrpcStatus.java

```diff
@@ -83,6 +84,47 @@ public static Status fromThrowable(Throwable t) {
         return s;
     }
 
+    /**
+     * Maps GRPC status codes to http status, as defined in upstream grpc-gateway
+     *
```

Comment:
nit: How about removing this empty line?

Reply:
+1